From 4b5a135fa681e85eb6988a379f59f3c7a41cc48c Mon Sep 17 00:00:00 2001
From: Gilles Talis <gilles.talis@gmail.com>
Date: Fri, 21 Jun 2013 15:25:11 -0700
Subject: [PATCH 1/2] Disable building test programs

We are not really interested in building test programs.
Moreover, these programs use fork() function that is
not available on MMU-less architectures.

Signed-off-by: Gilles Talis <gilles.talis@gmail.com>
[aduskett@gmail.com: Updated for 2.1.8]
Signed-off-by: Adam Duskett <aduskett@gmail.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 42879a3..dc90359 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -144,7 +144,6 @@
 BUILT_SOURCES =
 include include/include.am
 include sample/include.am
-include test/include.am
 
 if BUILD_WIN32
 
-- 
2.1.1

